ivy/functional/backends/torch/layers.py

```diff
@@ -674,3 +674,24 @@ def conv_general_transpose(
     if data_format == "channel_last":
         res = res.permute(0, *range(2, dims + 2), 1)
     return res
+
+
+conv_general_transpose.unsupported_dtypes = ("float16", "bfloat16")
+
+
+def dropout1d(
+    x: torch.Tensor,
+    prob: float,
+    /,
+    *,
+    training: bool = True,
+    data_format: str = "NWC",
+    out: Optional[torch.Tensor] = None,
+) -> torch.Tensor:
+    if data_format == "NWC":
+        perm = (0, 2, 1) if len(x.shape) == 3 else (1, 0)
+        x = torch.permute(x, perm)
+    res = torch.nn.functional.dropout1d(x, p=prob, training=training)
```

Comment:
our torch version is pinned to an older release, which doesn't have `dropout1d`, running the test locally throws an exception, you should implement it manually for now, adding a # ToDo to be updated with torch version is updated.